Automattic/wp-calypso
# Stats: Refactoring module grid

Related to https://github.com/Automattic/red-team/issues/6

## Proposed Changes

* simplify Traffic grid modules by moving modules or reducing their size to half width for easier stacking
* remove UTM module feature flag after the release

## Testing Instructions



* verify that the Traffic page grid works well with and without `stats/devices` feature flag
* test various blogs (with and without Jetpack)
* test various screen widths
* test enabling and disabling modules like Authors and Videos

Module view for versions that don't support it:
<img width="616" alt="SCR-20240506-nkxt" src="https://github.com/Automattic/wp-calypso/assets/112354940/2ca9cf0e-f998-4570-a1e7-3c9078e6e6d2">


## Pre-merge Checklist
